include/range/v3/utility/iterator_concepts.hpp

```diff
@@ -668,6 +668,9 @@ namespace ranges
 
         template<typename I, typename S = I>
         using sized_iterator_range_concept_t = meta::_t<sized_iterator_range_concept<I, S>>;
+
+        template<typename C>
+        using is_single_pass = meta::bool_<!DerivedFrom<C, ranges::forward_iterator_tag>()>;
```

Comment:
Only the ability to be used on the category alone, which is used by `postfix_increment_result` in basic_iterator.hpp. I didn't realize that `SinglePass` existed. I'll merge them.